buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
> Then, to say use '-s' for opening or deleting bookmarks doesn't look nice either.

`s` and `d` remain distinct. We are adding a **prompt key** to delete search results is all.